docs/compatibility.md

```diff
@@ -357,6 +357,21 @@ Also parsing of some values will not produce bit for bit identical results to wh
 They are within round-off errors except when they are close enough to overflow to Inf or -Inf which
 then results in a number being returned when the CPU would have returned null.
 
+### CSV ANSI day time interval
```

Comment:
nit: can we explain that this type was added in as a part of 3.3.0?

Reply:
done